i18n/en.json

```diff
@@ -203,7 +203,6 @@
   "admin.complianceExport.createJob.help": "Initiates a Compliance Export job immediately.",
   "admin.complianceExport.createJob.title": "Run Compliance Export Job Now",
   "admin.complianceExport.description": "This feature supports compliance exports to the Actiance XML and GlobalRelay EML formats, and is currently in beta. Support for the Mattermost CSV format is scheduled for a future release, and will replace the existing <a href=\"/admin_console/general/compliance\">Compliance</a> feature.",
-  "admin.complianceExport.description_without_globalrelay": "This feature supports compliance exports to the Actiance XML format, and is currently in beta. Support for GlobalRelay EML and the Mattermost CSV formats is scheduled for a future release, and will replace the existing <a href=\"/admin_console/general/compliance\">Compliance</a> feature.",
   "admin.complianceExport.exportFormat.actiance": "Actiance XML",
   "admin.complianceExport.exportFormat.actiance.description": "Format of the compliance export. Corresponds to the system that you want to import the data into. Compliance Export files will be written to the \"exports\" subdirectory of the configured <a href=\"/admin_console/files/storage\">Local Storage Directory</a>.",
```

Comment:
It's a bit confusing that the setting help text changes based on the setting value (different help text if Actiance or Global Relay selected).

Propose instead having a static help text with

```
Format of the compliance export. Corresponds to the system that you want to import the data into. 

For Actiance XML, compliance export files are written to the "exports" subdirectory of the configured [Local Storage Directory](link). For Global Relay EML, they are emailed to the configured email address.
```